### DISCUSSION TITLE
Unable to add my logging Handler to the default Task's logger

### DISCUSSION THREAD

**Discussion by sierrezinal** [PrefectHQ/prefect#3252] [Q&A] [answered]
```python
    from prefect.utilities.logging import get_logger

    rotater = RotatingFileHandler(str(root / 'output.log'), mode='a',
                                               maxBytes=5 * 1024 * 1024, backupCount=2,
                                               encoding='utf-8', delay=1)
    rotater.setLevel(logging.INFO)
    rotater.setFormatter(logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s'))
    task_logger = get_logger("Task")
    task_logger.addHandler(rotater)

    flow.run()
```

In my task

```python
@task
def transform(df: pd.DataFrame) -> pd.DataFrame:
    logger = prefect.context.get("logger")
    logger.info('blah blah')
```

I do see the output in stdout but no log file is created.

**Comment by cicdw** [comment, marked as answer]
Hi @sierrezinal - the task logger uses the name of your task as the logger name, so if you use
```python
task_logger = get_logger("transform")
```
I think it should work as you expect

**Reply by sierrezinal** [reply]
Aha. Will give it a spin.

P.S. Your example in the docs does not jive with your advice ?
https://docs.prefect.io/core/advanced_tutorials/custom-logs.html#deployment-logging

**Reply by cicdw** [reply]
Ah, thanks for letting me know!  PR to fix that doc here: https://github.com/PrefectHQ/prefect/pull/3253

**Reply by sierrezinal** [reply]
> Ah, thanks for letting me know! PR to fix that doc here: #3253

BTW everything is working now for my logging setup. Prefect rocks!
